File: CS/WindowsApplication2/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WindowsApplication2
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            Random r = new Random();
            for (int i = 0 ; i < 10; i++) {
                dataTable1.Rows.Add("name " + i.ToString(), r.NextDouble() * 100);
            }
        }

        private void gridView1_CustomDrawFooterCell(object sender, DevExpress.XtraGrid.Views.Grid.FooterCellCustomDrawEventArgs e)
        {
            if (e.Column.FieldName == "Amount") {
                object v = e.Info.Value;
                if (!(v is decimal)) {
                    return;
                }

                decimal d = Convert.ToDecimal(v);
                if (d <= 0)
                    e.Appearance.ForeColor = Color.Red;
                else if (d <= 500)
                    e.Appearance.ForeColor = Color.GreenYellow;
                else
                    e.Appearance.ForeColor = Color.Green;
            }
            
        }
    }
}

File: VB/WindowsApplication2/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace WindowsApplication2
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim r As New Random()
			For i As Integer = 0 To 9
				dataTable1.Rows.Add("name " & i.ToString(), r.NextDouble() * 100)
			Next i
		End Sub

		Private Sub gridView1_CustomDrawFooterCell(ByVal sender As Object, ByVal e As DevExpress.XtraGrid.Views.Grid.FooterCellCustomDrawEventArgs) Handles gridView1.CustomDrawFooterCell
			If e.Column.FieldName = "Amount" Then
				Dim v As Object = e.Info.Value
				If Not(TypeOf v Is Decimal) Then
					Return
				End If

				Dim d As Decimal = Convert.ToDecimal(v)
				If d <= 0 Then
					e.Appearance.ForeColor = Color.Red
				ElseIf d <= 500 Then
					e.Appearance.ForeColor = Color.GreenYellow
				Else
					e.Appearance.ForeColor = Color.Green
				End If
			End If

		End Sub
	End Class
End Namespace